packages/core/contracts/data-verification-mechanism/implementation/Staker.sol

```diff
@@ -291,10 +291,10 @@ abstract contract Staker is StakerInterface, Ownable, Lockable, MultiCaller {
      * @return address voter that corresponds to the delegate.
      */
     function getVoterFromDelegate(address caller) public view returns (address) {
-        if (
-            delegateToStaker[caller] != address(0) && // The delegate chose to be a delegate for the staker.
-            voterStakes[delegateToStaker[caller]].delegate == caller // The staker chose the delegate.
-        ) return delegateToStaker[caller];
+        address delegateToStaker = delegateToStaker[caller];
```

Comment:
Should this variable better be renamed to `delegator` for clarity?

Reply:
yup! good catch, have changed.